rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/tests/activityList.spec.ts

```diff
@@ -35,7 +35,7 @@ const shallowMountActivityList = async (
         Btn,
       },
       stubs: {
-        // modal: false,
+        modal: false,
         btn: false,
         "i18n-t": true,
         ProgressBar: true,
```

Comment:
@smartinellibenedetti Keeping` i18n-t": true` and `ProgressBar: true`  helps avoiding console warnings  Test is passing after I removed it but getting console warnings.